build-constraints.yaml

```diff
@@ -364,8 +364,7 @@ packages:
         - gravatar
 
     "Paul Harper <benekastah@gmail.com>":
-        []
-        # GHC 8 - yesod-auth-oauth2
+        yesod-auth-oauth2
```

Comment:
Please change this to

    - yesod-auth-oauth2

(This section must be a yaml list, not just a single string)